georgeforge/app_settings.py

```diff
@@ -3,7 +3,10 @@
 # Django
 from django.conf import settings
 
-# put your app settings here
 
+def discord_bot_active():
+    return "aadiscordbot" in settings.INSTALLED_APPS
 
-# EXAMPLE_SETTING_ONE = getattr(settings, "EXAMPLE_SETTING_ONE", None)
+FORGE_MARKET_GROUPS = getattr(settings, "FORGE_MARKET_GROUPS", [4,6,7,8,18,20,63,66])
```

Comment:
are these categories or market_groups also we should probably make sure we preload them instead of the hardcoded ones. I can look at doing this later